slime/utils/arguments.py

```diff
@@ -76,11 +76,20 @@ def add_cluster_arguments(parser):
                 ),
             )
             parser.add_argument(
-                "--offload",
```

Comment:
can we still leave the old `--offload`, where it will enable both `offload_train` and `offload_rollout` for backward compatible?

Reply:
sure